ci/jenkins/jobs.yaml

```diff
@@ -15,25 +15,18 @@
         - '{name}-jjb-validation'
 
 - project:
-      name: caasp-jobs/caasp-v4-openstack
+      name: caasp-jobs/v4/conformance
       repo-name: skuba
       repo-owner: SUSE
       repo-credentials: github-token
-      platform: openstack
-      jobs:
-          - '{name}-conformance'
-
-- project:
-      name: caasp-jobs/caasp-v4-vmware
-      repo-name: skuba
-      repo-owner: SUSE
-      repo-credentials: github-token
-      platform: vmware
+      platform:
+        - openstack
+        - vmware
       jobs:
-          - '{name}-conformance'
+          - '{name}-{platform}-conformance'
```

Comment:
You have to modify the template for conformance tests and make name match `{name}-{platform}-{conformance}

https://github.com/SUSE/skuba/blob/6b2bb427946eeeae97bb7c723e77b29f8a3679f9/ci/jenkins/templates/conformance-template.yaml#L2

Reply:
right, just saw the failure on jjb check. Thanks for the hint, lets see if now the test pass.